docker-compose.yaml

```
@@ -0,0 +1,60 @@
+version: "3"
+
+services:
+  graphdb:
+    image: arangodb:3.8.5
+    platform: linux/amd64
+    environment:
+      - ARANGO_ROOT_PASSWORD=
+    ports:
+      - 8529:8529
+    volumes:
+      - resoto_graphdb_data:/var/lib/arangodb3
+  resotocore:
+    image: ghcr.io/someengineering/resotocore
+    depends_on:
+      - graphdb
+    ports:
+      - "8900:8900/tcp"
+    environment:
+      - PSK
+      - RESOTOCORE_GRAPHDB_SERVER=http://graphdb:8529
+      - RESOTOCORE_GRAPHDB_PASSWORD=changeme
+    restart: always
```

```suggestion
    restart: on-failure[:max-retries]
```
It may be better to set a number of retries to prevent a restart loop.